CHANGES.rst

```diff
@@ -19,7 +19,12 @@ Release History
    - Removed
    - Fixed
 
-0.1.0 (unreleased)
+0.1.0 (12/06/2016)
```

Comment:
In other Nengo projects we use the date format Month DD, YYYY (e.g. December 6, 2016) as it tends to be easier to read when scanning down a list of releases... dunno how important consistency is there, but might as well be consistent I guess?